.github/actions/install-dependencies/install_windows_static_x86_64.sh

```
@@ -0,0 +1,71 @@
+set -e
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)